src/session.ts

```diff
@@ -175,20 +187,25 @@ class Session extends ServiceObject {
        */
       id: name,
       methods,
-      createMethod: (_, options, callback) => {
-        if (is.fn(options)) {
-          callback = options;
-          options = {};
-        }
-        return database.createSession(options, (err, session, apiResponse) => {
-          if (err) {
-            callback(err, null, apiResponse);
-            return;
-          }
+      // tslint:disable-next-line no-any
```

Comment:
is this comment needed here?

Reply:
for ignoring any error in linting

Reply:
oh, Didn't know this comment is for linting, That's fine is then.